dockers/docker-base-stretch/sources.list

```diff
@@ -6,3 +6,4 @@ deb-src http://debian-archive.trafficmanager.net/debian/ stretch main contrib no
 deb http://debian-archive.trafficmanager.net/debian-security/ stretch/updates main contrib non-free
 deb-src http://debian-archive.trafficmanager.net/debian-security/ stretch/updates main contrib non-free
 deb http://debian-archive.trafficmanager.net/debian/ stretch-backports main contrib non-free
+deb http://debian-archive.trafficmanager.net/debian/ unstable main contrib non-free
```

Comment:
buster has the same package, let's use buster.

https://packages.debian.org/search?suite=buster&keywords=python-scapy